sources/packages/web/src/main.ts

```diff
@@ -27,6 +27,7 @@ import ToggleContent from "@/components/generic/ToggleContent.vue";
 import FormioContainer from "@/components/generic/FormioContainer.vue";
 import FooterButtons from "@/components/generic/FooterButtons.vue";
 import TitleValue from "@/components/generic/TitleValue.vue";
+import HorizontalSeparator from "@/components/generic/HorizontalSeparator.vue";
```

Comment:
what is this used for?

Reply:
That was a component already in the code. I will replace it but the v-divider.

Reply:
Component removed.